docs/resources/dashboard.md

```diff
@@ -13231,6 +13241,14 @@ Optional:
 
 
 
+<a id="nestedblock--widget--sunburst_definition--request--style"></a>
+### Nested Schema for `widget.sunburst_definition.request.style`
+
+Optional:
+
+- `palette` (String) A color palette to apply to the widget. The available options are available at: https://docs.datadoghq.com/dashboards/widgets/timeseries/#appearance.
```

Comment:
**question:** Is it intentional that we're linking to the timeseries docs here? If these docs are generated then we may need to fix it upstream

Reply:
Looks like most of the widget request styles share the same schema from `getWidgetRequestStyle()` line 9192 of `resource_datadog_dashboard.go`. Is there a more generic list of styles that could be linked to?

Reply:
Update: it's on their radar. They're going to do a review of color docs, as I think there are a few more stale things beyond just this.